hack/local-up-cluster.sh

```diff
@@ -51,7 +52,7 @@ EVICTION_PRESSURE_TRANSITION_PERIOD=${EVICTION_PRESSURE_TRANSITION_PERIOD:-"1m"}
 # container bridge docker is currently using) and we don't know the IP of the
 # DNS pod to pass in as --cluster-dns. To set this up by hand, set this flag
 # and change DNS_SERVER_IP to the appropriate IP.
-ENABLE_CLUSTER_DNS=${KUBE_ENABLE_CLUSTER_DNS:-false}
+ENABLE_CLUSTER_DNS=${ENABLE_CLUSTER_DNS:-false}
```

Comment:
I was annoyed that they didn't match and our other enables aren't KUBE prefixed.  I can revert though.